scripts/parser-tests/test262/allowlist.txt

```diff
@@ -1,17 +1,14 @@
-# 0 invalid programs did not produce a parsing error
+# 5 invalid programs did not produce a parsing error
 
+language/expressions/assignmenttargettype/direct-callexpression-in-logical-assignment.js(default)
+language/expressions/assignmenttargettype/parenthesized-callexpression-in-logical-assignment.js(default)
+language/expressions/logical-assignment/lgcl-and-assignment-operator-non-simple-lhs.js(default)
+language/expressions/logical-assignment/lgcl-nullish-assignment-operator-non-simple-lhs.js(default)
+language/expressions/logical-assignment/lgcl-or-assignment-operator-non-simple-lhs.js(default)
```

Comment:
It seems impossible to distinguish them with the information available.
I noticed that the engine262 tests at https://github.com/tc39/test262/pull/4459 also fail.
Do we need to ask in test262?

Reply:
Yes, the cases involving the logical assignment operators remain syntax errors.
(See https://github.com/tc39/ecma262/pull/2193#issuecomment-716536425 for the origin story there.)